lsst_efd_client/efd_helper.py

```diff
@@ -377,9 +418,67 @@ async def select_packed_time_series(self, topic_name, base_fields, start, end,
         result = await self.select_time_series(topic_name, field_list+[ref_timestamp_col, ],
                                                start, end, is_window=is_window, index=index)
         vals = {}
-        for field in base_fields:
-            df = merge_packed_time_series(result, field, ref_timestamp_col=ref_timestamp_col,
+        for f in base_fields:
+            df = merge_packed_time_series(result, f, ref_timestamp_col=ref_timestamp_col,
                                           internal_time_scale=self.internal_scale)
-            vals[field] = df[field]
+            vals[f] = df[f]
         vals.update({'times': df['times']})
         return pd.DataFrame(vals, index=df.index)
+
+    async def get_schema_topics(self):
```

Comment:
My suggestion for the interface is to have a `EFDClient.get_topics()` method.

In principle, the list of topics retrieved from the Schema Rregistry should match the list of measurements in influxDB. But I think we should provide only one method to retrieve the list of topics and my preference would be the result of "SHOW MEASUREMENTS", which ultimately is the data available to query.

Reply:
So is it guaranteed that the topics returned by "SHOW MEASUREMENTS" will be in the schema registry?

Reply:
OK.  Removed redundant get_topics